projects/karchive/Dockerfile

```diff
@@ -15,16 +15,14 @@
 ################################################################################
 
 FROM gcr.io/oss-fuzz-base/base-builder
-RUN apt-get update && apt-get install --yes cmake make autoconf automake autopoint libtool wget po4a
+RUN apt-get update && apt-get install --yes cmake make autoconf automake autopoint libtool wget po4a ninja-build pkgconf libzstd-dev
```

Comment:
Installing libzstd-dev is wrong, this distribution provided library is not instrumented and thus it's unusable, if zstd is a new dependency it needs to be compiled like all the others

Reply:
We can probably go with -DWITH_LIBZSTD=OFF for now for karchive cmake since the goal of this is port to Qt6